<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrinterBehavior</class>
 <widget class="QWidget" name="PrinterBehavior">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" rowspan="2" colspan="5">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Banners</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" rowspan="2" colspan="5">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="4">
    <widget class="QRadioButton" name="allowRB">
     <property name="text">
      <string>Allow these users to print</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="5">
    <widget class="QRadioButton" name="preventRB">
     <property name="text">
      <string>Prevent these users from printing </string>
     </property>
    </widget>
   </item>
   <item row="17" column="0" colspan="5">
    <widget class="KEditListBox" name="usersELB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="buttons">
      <set>KEditListBox::Add|KEditListBox::Remove</set>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Ending Banner:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>endingBannerCB</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Starting Banner:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>startingBannerCB</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="KComboBox" name="endingBannerCB">
     <property name="AttributeName" stdset="0">
      <string notr="true">job-sheets-default</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="KComboBox" name="startingBannerCB">
     <property name="AttributeName" stdset="0">
      <string notr="true">job-sheets-default</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="5">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Operation Policy:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>operationPolicyCB</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Error Policy:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>errorPolicyCB</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KComboBox" name="operationPolicyCB">
     <property name="AttributeName" stdset="0">
      <string notr="true">printer-op-policy</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KComboBox" name="errorPolicyCB">
     <property name="AttributeName" stdset="0">
      <string notr="true">printer-error-policy</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0" colspan="5">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Allowed Users</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Policies</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
